src/test/java/net/sf/jabref/logic/importer/fetcher/MrDLibFetcherTest.java

```diff
@@ -0,0 +1,40 @@
+package net.sf.jabref.logic.importer.fetcher;
+
+import java.util.List;
+
+import net.sf.jabref.logic.importer.FetcherException;
+import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.model.entry.FieldName;
+import net.sf.jabref.testutils.category.FetcherTests;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import static org.junit.Assert.assertEquals;
+
+@Category(FetcherTests.class)
+public class MrDLibFetcherTest {
+
+    private MrDLibFetcher fetcher;
+    private BibEntry bibEntry;
+
+    @Before
+    public void setUp() {
+        fetcher = new MrDLibFetcher("", "");
+        bibEntry = new BibEntry();
+        bibEntry.setField(FieldName.TITLE, "lernen");
+    }
+
+    @Test
+    public void testPerformSearch() throws FetcherException {
+        List<BibEntry> bibEntrys = fetcher.performSearch(bibEntry);
+        assertEquals(false, bibEntrys.isEmpty());
```

Comment:
They are not unpredictable. 
We are constantly extending our database. This means new documents can be recommended. Though the output of the server can be changed. 

The test above does a server request. So how can I find the result of the ci-server ?